lib/UI.rb

```diff
@@ -88,12 +78,12 @@ def prompt_draw
     puts 'Game draw!'
   end
 
-  def prompt_invalid_space
-    puts 'Please enter a valid space'
+  def prompt_invalid_input(lower_int, upper_int)
+    puts "Please enter a valid selection between #{lower_int} and #{upper_int}"
   end
 
-  def prompt_invalid_input
-    puts 'Please enter a number from 1-9'
+  def prompt_invalid_space
+    puts "Please enter a valid move"
```

Comment:
There seem to be a few different types of error prompts that do the same job. "Please enter a valid selection between int and in" and "Please enter a valid move". I personally like the more descriptive former than the latter.